### ISSUE TITLE
sChinese localization

### ISSUE THREAD

**Issue by stackia** [Lange/SOAP-TF2DM#10]
```c++
"Phrases"
{
	"Health Remaining"
	{
		"#format"		"{1:i}"
		"chi"			"您的对手还剩 {1} 点生命。"
	}
	"Attacker is dead"
	{
		"chi"			"您的对手死了。"
	}
	"Plugins unloaded"
	{
		"chi"			"插件已卸载。"
	}
	"Plugins reloaded"
	{
		"chi"			"插件已重新载入。"
	}
}
```

**Comment by stackia** [comment]
Updated.
```
"Phrases"
{
	"Health Remaining"
	{
		"chi"			"对手还剩 {1} HP。"
	}
	"Attacker is dead"
	{
		"chi"			"对手死掉了。"
	}
	"Plugins unloaded"
	{
		"chi"			"插件已卸载。"
	}
	"Plugins reloaded"
	{
		"chi"			"插件已重载。"
	}
	"Kill HP Received"
	{
		"chi"			"你杀死了 {2}，因此获得 {1} HP"
	}
	"Damage HP Received"
	{
		"chi"			"你对 {2} 造成了伤害，因此获得 {1} HP"
	}
}
```

**Comment by Lange** [comment]
Can you open a pull request with this? Thanks!

**Comment by stackia** [comment]
https://github.com/Lange/SOAP-TF2DM/pull/27

**Comment by Lange** [comment]
Resolved in #27
